packages/db/db-client-types.ts

```diff
@@ -218,6 +219,8 @@ export type UploadItemOutput = {
   dagSize?: definitions['content']['dag_size']
   pins: Array<PinItemOutput>,
   deals: Array<Deal>
+  // array of links to things containing this Upload (e.g. CARs)
+  partOf: Array<string>
```

Comment:
Maybe just pluralise as `parts`? I like that this is similar to a [partition content claim](https://github.com/web3-storage/content-claims#partition-claim).

Reply:
Will do.

Just to check my understanding. Do we ever expect there to be more than one CAR that the Upload is sharded across? In my light testing it looked like only CARs with all blocks included. So I was thinking we could make a slightly different claim than just 'it's spread across these parts' but instead 'it's entirely within this CAR, i.e. it is a subset/partOf these CARs'. But I did switch it to `parts` because that's a less opinionated claim here that might fit the data better.

Reply:
It can be multiple. If they send another CAR with the same root CID it _should_ be added to the list.